package.json

```diff
@@ -23,7 +23,8 @@
     "pg": "^4.4.3",
     "ramda": "^0.19.0",
     "request": "^2.67.0",
-    "slack-client": "^1.5.1"
+    "slack-client": "^1.5.1",
+    "wordwrap": "^1.0.0"
```

Comment:
How did it get added in the PR without the corresponding digest.js code? Was it accidentally left out of a prior PR?